src/battle-scene.ts

```diff
@@ -889,6 +894,13 @@ export default class BattleScene extends SceneBase {
     this.infoToggles.push(infoToggle);
   }
 
+  removeInfoToggle(infoToggle: InfoToggle): void {
+    const infoToggleIndex = this.infoToggles.indexOf(infoToggle);
+    if (infoToggleIndex !== -1) {
+      this.infoToggles.splice(infoToggleIndex, 1);
+    }
+  }
+
```

Comment:
NIT: 
1. Missing scope `public`
2. Missing tsdoc

-----

```suggestion
  /**
   * Removes a specified {@linkcode InfoToggle} from the {@linkcode infoToggles} array.
   *
   * @param infoToggle - The {@linkcode InfoToggle} instance to be removed. If the toggle
   *                     is not found in the array, no action is taken.
   */
  public removeInfoToggle(infoToggle: InfoToggle): void {
    const infoToggleIndex = this.infoToggles.indexOf(infoToggle);
    if (infoToggleIndex !== -1) {
      this.infoToggles.splice(infoToggleIndex, 1);
    }
  }

```